data/sql_setup/prepare_schedule_a.sql

```diff
@@ -101,13 +48,6 @@ begin
                 delete from ofec_sched_a_queue_new where sub_id = view_row.sub_id;
                 insert into ofec_sched_a_queue_new values (view_row.*, timestamp, two_year_transaction_period);
             end if;
-        else
-            -- We weren't able to successfully retrieve a row from the view,
-            -- so keep track of this sub_id if we haven't already so we can
-            -- try processing it again each night until we're able to
-            -- successfully process it.
-            delete from ofec_sched_a_nightly_retries where sub_id = new.sub_id;
-            insert into ofec_sched_a_nightly_retries values (new.sub_id, 'insert');
         end if;
 
         return new;
```

Comment:
I was confused by this and had to RTFM to make sure I understood what is being returned. It may be better to use:
```
 RETURN NULL; -- result is ignored since this is an AFTER trigger
```
 as shown in an example on https://www.postgresql.org/docs/9.6/static/plpgsql-trigger.html#PLPGSQL-DML-TRIGGER